index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css"/>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
+  </head>
+  <body>
+    <main>
+      <h1 class="header">Hello Microverse</h1>
```

Comment:
**Optional:** consider using class names that are unique. In this case, you created a class `header` which could be confused with `<header>` tags.

You could use instead a class `title` for example.